### ISSUE TITLE
User anime lists missing "Tags" filter, What should I do?

### ISSUE THREAD

**Issue by JoNatthanon** [zend10/AL-chan#150]
The tags filter option is not present on the "my collection" anime lists tab. See screen recording below.

https://github.com/zend10/AL-chan/assets/142714590/3a1f689c-a6ef-4ba3-8a61-edaeee7b0482

Therefore, I ask the admin to update the new version to 2.9.0+ to fix the problem of tag filter in the user anime list that has disappeared and has returned again.  And will it be released to the Google Play Store asap? (with link)

Or, going back to using the old version 1.2.8 would be best.
@zend10 @TheAkio @infanf @axiel7

**Comment by peepincreepin** [comment]
I have just realised from [this](https://github.com/zend10/AL-chan/releases/tag/v2.1.0) that it's probably because you have over 1500 entries in your anime list.

> Remove Tag filter and Releasing Today for users with 1500 entries in Media List.

I have the same thing in my manga list which has 1900 entries. It's really restrictive. I assume it was implemented because of rate limit issues, but I hope the app dev increases the limit from 1500 or removes it entirely. There are other apps which don't have this limit and still work fine, so I would hope that it's doable.
